xml/geo_manage_i.xml

```diff
@@ -29,8 +29,20 @@
  <sect1 xml:id="sec.ha.geo.manage.cli">
   <title>Managing Tickets From Command Line</title>
 
+  <remark>toms 2018-06-14: there is no "booth client" subcommand. According
```

Comment:
I think "client" is basically the default subcommand, meaning unless the special subcommands like "daemon" are supplied,  "booth ..." works same as "booth client", @ChrisKowalczyk?

Reply:
Hmn, it's a bit confusing. According to the booth man page:

```
SYNOPSIS
       boothd daemon [-SD] [-c config] [-l lockfile]
       booth list [-s site] [-c config]
       booth grant [-s site] [-c config] [-FCw] ticket
       booth revoke [-s site] [-c config] [-w] ticket
       booth peers [-s site] [-c config]
       booth status [-D] [-c config]
```

However, if you look a little bit deeper, I found this:

```
COMMANDS
     Whether the binary is called as boothd or booth doesn’t matter; the first
     argument determines the mode of operation.

       daemon
           Tells boothd to serve a site. The locally configured interfaces are searched for an IP address
           that is defined in the configuration. booth  then runs in either /arbitrator/ or /site/ mode.

       client
           Booth clients can list the ticket information (see also crm_ticket -L), and revoke or grant tickets to a site.
           [...]
```

Well, it could be debated if the man page is inconsistent or structured unfortunate. I assume, to get an overview of all `booth` subcommands you would probably look into the synopsis first. 

Maybe it's just me, but from my point of view, the synopsis of the man page is incomplete.

Reply:
The manual is indeed confusing. I guess "client" is not listed in "SYNOPSIS" because it's no longer recommended/needed, given that all the subcommands supported by the client, such are "list", "grant" and so on are listed there already and can be directly invoked. I'd  just skip "client" in our doc. Opinions, @ChrisKowalczyk?

Reply:
Agreed. I have never used "booth client". Let us skip it!